orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -409,9 +414,23 @@ def _get_exported_model(self, input_schema: ORTModelInputOutputSchemaType, *inpu
             )
         exported_model = onnx.load_model_from_string(f.getvalue())
 
-        exported_model = _post_process_after_export(
-            exported_model, self._runtime_options.enable_custom_autograd_function
-        )
+        if self._runtime_options.enable_custom_autograd_function:
+            from ._custom_autograd_function_exporter import post_process_enabling_autograd_function
+
+            exported_model = post_process_enabling_autograd_function(exported_model)
+
+        if self._runtime_options.enable_zero_stage3_support:
+            from ._zero_stage3_compatibility import post_processing_enable_zero_stage3_compat
+
+            exported_model = post_processing_enable_zero_stage3_compat(
+                exported_model,
+                self._zero_stage3_param_map,
+                [name for name, _ in self._flattened_module.named_parameters()],
+            )
+
+            # Cannot append pull weight trigger name to input names here, otherwise, the later check find
+            # input info mismatch, will re-initialize the graph builder.
+            # self._input_info.require_grad_names.append(STAGE3_PULL_WEIGHT_TRIGGER_NAME)
```

Comment:
Is this comment here intentional?

Reply:
Yes, this is by intention in case we later append STAGE3_PULL_WEIGHT_TRIGGER_NAME in self._input_info.require_grad_names, after model export, `_reinitialize_graph_builder` will check self._input_info with a newly generated input_info, if they don't match, the will re-export. 

I have add more details in the comments for clarification.